src/pages/Swap.tsx

```diff
@@ -128,11 +127,11 @@ function Swap(): ReactElement {
         to: [],
       }
 
-    const allTokens = Object.values(TOKENS_MAP)
-      .filter(({ isLPToken, addresses }) => !isLPToken && addresses[chainId])
+    const allTokens = Object.values(tokens || {})
+      .filter(({ isLPToken, address }) => !isLPToken && address)
```

Comment:
you can remove `address` here, the old logic was just making sure the token existed on the current chain, but `tokens` does that automatically.